rules/crowdstrike_rules/crowdstrike_suspicious_cron.yml

```diff
@@ -0,0 +1,72 @@
+AnalysisType: rule
+Filename: crowdstrike_suspicious_cron.py
+RuleID: "Crowdstrike.SuspiciousCron"
+DisplayName: "Crowdstrike - Suspicious cron detected"
+Enabled: true
+LogTypes:
+  - Crowdstrike.FDREvent
+Tags:
+  - Crowdstrike.FDREvent
+  - Execution:Scheduled Task/Job
+Reports:
+  MITRE ATT&CK:
+    - TA0002:T1053
+Severity: High
+Description: A suspicious cron has been added
+Runbook: Analyze the command to ensure no nefarious activity is occurring
+Reference: https://en.wikipedia.org/wiki/Cron
+SummaryAttributes:
+  - aid
+  - p_any_ip_addresses
+Tests:
```

Comment:
How are you testing against a cron being added with one of these commands? It seems like its only testing if the `wget` or `nc` command is being used on the `CommandLine` field?